README.md

```diff
@@ -1,73 +1,72 @@
-# Opdrachtbeschrijving
+# Opdrachtbeschrijving-test
```

Comment:
De wijzigingen in README.md zijn goed en geven een duidelijke uitleg van de applicatie. Het is belangrijk dat je consistent blijft in je uitleg en terminologie. Overweeg om de uitleg over de databankverbindingen te verduidelijken.